nano/node/json_handler.cpp

```diff
@@ -913,11 +914,18 @@ void nano::json_handler::accounts_balances ()
 			balances.put_child (account_from_request.second.data (), entry);
 			continue;
 		}
-		entry.put ("error", ec.message ());
-		balances.put_child (account_from_request.second.data (), entry);
+		debug_assert (ec);
+		errors.put (account_from_request.second.data (), ec.message ());
 		ec = {};
 	}
-	response_l.add_child ("balances", balances);
+	if (!balances.empty ())
```

Comment:
@pwojcikdev , when approving both RPC PRs, they both should come with the same solution for empty entries.

Reply:
I think it's the best workaround for now, added the same logic to my PR